PROTOCOL.md

```diff
@@ -424,6 +429,66 @@ enough to get you started. You can watch the data sent and received from
 the server on a regular connection, or look at the client source code
 for a full list of message types.
 
+**Action requests**
+
+These are how the client sends the player's decisions to the server. Most
+requests can be sent with `|RQID`. `RQID` is a unique number used to
+identify which action the request was intended for and is used to protect
+against race conditions with the cancel button.
+
+If an invalid request is sent, the game will replace the missing or
+erroneous request with the first valid choice, which is usually the first
+usable move, or Struggle if there are no usable moves left. Similarly, if
+a switch was expected but a move was sent instead, the game will switch in
+the first usable Pokemon.
+
+`|/team ORDER`
+
+> Chooses the team order. Numbers not listed are displaced to the back by swapping
+> them with the number that took their place. For example `/team 25` sets the team
+> order from default to 253416.
+
+`|/move NUMBER TARGET`
+
+> Requests to use the selected move `NUMBER` to be used on `TARGET` Pokemon. `NUMBER`
+> starts at 1 and can be up to 12 (though usually 4), where `NUMBER` calls the move
+> indexed at `NUMBER - 1` in the Pokemon's set.
```

Comment:
I agree that I haven't phrased that very well but I think it would be good to give some indication of how the moves are ordered  in relation to the number being sent, just to be overly clear that moves aren't sorted alphabetically or anything like that. I'm not sure how to put that into words though.